tasks/coverage/src/typescript/constants.rs

```diff
@@ -206,7 +206,195 @@ pub static NOT_SUPPORTED_ERROR_CODES: phf::Set<&'static str> = phf::phf_set![
     "2537", // Type '{ bar: string; }' has no matching index signature for type 'string'.
     "2538", // Type '[]' cannot be used as an index type.
     "2540", // Cannot assign to 'ro' because it is a read-only property.
+    "2542", // Index signature in type 'DeepReadonlyArray<Part>' only permits reading.
+    "2545", // A mixin class must have a constructor with a single rest parameter of type 'any[]'.
+    "2548", // Type 'number' is not an array type or does not have a '[Symbol.iterator]()' method that returns an iterator.
+    "2550", // Property 'setBigInt64' does not exist on type 'DataView<ArrayBuffer>'. Do you need to change your target library? Try changing the 'lib' compiler option to 'es2020' or later.
+    "2551", // Property 'fng2' does not exist on type 'typeof A'. Did you mean 'fng'?
+    "2552", // Cannot find name 'tupel'. Did you mean 'tuple'?
+    "2554", // Expected 0 arguments, but got 1.
+    "2555", // Expected at least 1 arguments, but got 0.
+    "2556", // A spread argument must either have a tuple type or be passed to a rest parameter.
+    "2558", // Expected 0 type arguments, but got 1.
+    "2559", // Type 'D' has no properties in common with type 'C'.
+    "2560", // Value of type '() => { timeout: number; }' has no properties in common with type 'Settings'. Did you mean to call it?
+    "2561", // Object literal may only specify known properties, but 'colour' does not exist in type 'CSSProps'. Did you mean to write 'color'?
+    "2562", // Base class expressions cannot reference class type parameters.
```

Comment:
```
    class Gen<T> extends base<T>() {}  // Error, T not in scope
                              ~
!!! error TS2562: Base class expressions cannot reference class type parameters.
```
On second though, should this be not included in this list?

Reply:
When we're unsure, just leave it as it is. 🙈